### PR TITLE
US19311 Logged In Snail trail

### PR DESCRIPTION
**Pull request by silasbaur**
## Task
- update crds-data with the new snail-trail
- have the links from the snail trail open nav trays

[Rally Story](https://rally1.rallydev.com/#/38108142417d/detail/userstory/379482156500?fdp=true)

### Additional Task
We created a Utility that can get Graphql info that comes from crds-data. The utility looks for tokenized strings and grabs the content in between the "%", which in this case would be `site.name` and pass that through Graphql to get the users site name.
`"What's Happening%\" at \"site.name%"`

## Corresponding PR 
[Crds-data](https://github.com/crdschurch/crds-data/pull/37)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -55,7 +55,7 @@
   </div>
 
   <div class="snail-trail-skeleton">
-    <crds-snail-trail name="media" env="int"></crds-snail-trail>
+    <crds-snail-trail name="logged-in" env="int"></crds-snail-trail>
```
Comment (dholloran-ample) [inline src/index.html:58]: How will this work once it gets into crds-net, crds-media, etc.? Will we need to update anything else to make this work?

**Comment by dholloran-ample** [comment]
@silasbaur,

One more thing I noticed is the old menu supposed to be removed when logged out (this may be something we can ask in refinement if you don't know)?

CleanShot 2020-04-28 at 10 07 40

File: src/index.html (hunk 1 above)
Comment (silasbaur) [reply src/index.html:67]: @dholloran-ample  This is only being used on crds-net on the homepage when you are logged in. In [this file](https://github.com/crdschurch/crds-net/blob/master/logged-in-homepage.html) we would add this to the top section `snail_trail: logged-in` to handle what snail trail is shown.

Comment (dholloran-ample) [reply src/index.html:67]: @silasbaur - Ok that makes sense I spaced on the fact that its home page only

**Pull request merged**